### ISSUE TITLE
TypeError when a local file is set

### ISSUE THREAD

**Issue by nekki-t** [ascoders/react-native-image-viewer#179]
Hi there,

I'm having a problem regarding with loading a local image.
I've been trying to do it by writing code as follows:
```
<ImageViewer
    imageUrls={[{ props: { source: require('../assets/images/Sample.png') } }]}
/>
```

Then, I'm receiving errors as in the image below.
![local-file](https://user-images.githubusercontent.com/13582190/39566618-2d158d96-4ef7-11e8-91c1-8b82b15fe1d1.png)

Please kindly help me solve this problem.

**Comment by ascoders** [comment]
Maybe it's not a bug. See the warning line: `This error is a performance optimization and not spec compliant`.

**Comment by nekki-t** [comment]
Thanks!

I don't think it is a bug, either, but an optimization issue.
Besides, do you have any suggestions to clear this problem out?

**Comment by ascoders** [comment]
I want to clear it too~, anyone help?

**Comment by ascoders** [comment]
Thanks @zamponotiropita ! I fixed it in `2.2.4`.

**Comment by zamponotiropita** [comment]
@ascoders great, it works now!

**Comment by zamponotiropita** [comment]
@ascoders in `2.2.4` first image is not shown now (black)

**Comment by ascoders** [comment]
Hi, @zamponotiropita , can you give an example?
I can't reproduce in my environment.

**Comment by ascoders** [comment]
Sorry, it's my mistake, fixed in `2.2.5`.

**Comment by zamponotiropita** [comment]
@ascoders great! thanks for your quick fixes :D
